### DISCUSSION TITLE
Refactoring the dmd ast

### DISCUSSION THREAD

**Discussion by tripleo1** [dlang/dmd#16228] [Show and tell]
ref: https://github.com/dlang/dmd/blob/master/CONTRIBUTING.md#how-you-can-help

Why don't you write a program to do this?

> Celebrate Your Contribution: Take pride in being a part of this significant compiler development effort!

I put this in `Show and tell` to showcase the fact that I am not putting my money where my mouth is.

**Comment by dkorpel** [comment]
Writing such a program seems more work than doing the refactoring manually, but I invite anyone to try.
